blocks/cards/cards.css

```diff
@@ -191,6 +191,18 @@ main .section.cards-container:has(.cards.info-points) h2 {
   margin-bottom: 2rem;
 }
 
+.cards.grid .cards-card-image img {
+  object-position: center;
+}
+
+.cards.grid .cards-card-body .title {
+  display: -webkit-box;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
```

Comment:
Do we need vendor specific properties?

Reply:
@sdmcraft Definitely, I think we need those properties. Do you have any suggestions?
https://developer.mozilla.org/ja/docs/Web/CSS/-webkit-line-clamp

Reply:
I wasn't able to find `-webkit-box` property documented anywhere. Could you tell what it does? Similarly for `webkit-box-orient`.
Also how about using `line-clamp:` instead of `-webkit-line-clamp:`

Reply:
@sdmcraft About `-webkit-box`, yes, I can only find some related description from https://developer.mozilla.org/en-US/docs/Web/CSS/box-flex

And, `line-clamp` doesn't work, you can see here.
<img width="286" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/7f7149fa-5245-4899-b606-5de93694a3df">